### ISSUE TITLE
Buyers' guide landing page updates

### ISSUE THREAD

**Issue by rosamundtgov** [GSA/oasis-plus#412]

**Comment by klohman** [comment]
## Updated design for OASIS+ buyers' ordering guide (printable) page 

- [Figma mockup](https://www.figma.com/file/lWfZDygV7WbvGlea2PIrpa/OASIS-Playground?type=design&node-id=3743-32438&mode=design&t=5es2QtqI6kc2N3Qe-4) 

@mark-meyer @weiwang-gsa I did not change the header sizes in the mockup - so we may need to discuss how to translate headings so we only have one h1 on this page.

**Comment by rosamundtgov** [comment]
waiting on feedback from @PowderD and @ekidenda. @klohman there have been some text updates, let's discuss.

**Comment by ekidenda** [comment]
@rosamundtgov review done and feedback provided on the content doc.

**Comment by rosamundtgov** [comment]
> ## Updated design for OASIS+ buyers' ordering guide (printable) page
> * [Figma mockup](https://www.figma.com/file/lWfZDygV7WbvGlea2PIrpa/OASIS-Playground?type=design&node-id=3743-32438&mode=design&t=5es2QtqI6kc2N3Qe-4)
> 
> @mark-meyer @weiwang-gsa I did not change the header sizes in the mockup - so we may need to discuss how to translate headings so we only have one h1 on this page.

This has been separated into another ticket. #449

**Comment by jfredrickson** [comment]
Note that the Research agencies and sellers link indicated in the content depends on #448, which is still in the backlog. Will need to add the link at a later time.

**Comment by rosamundtgov** [comment]
@jfredrickson Update the description under Complete market research with the following:

Explore the federal market; find data on business utilization, spending and use tools, and opportunities for subcontracting and joint ventures. As part of this, learn about pricing intelligence, scope reviews, and using Symphony Procurement Suite or the free Market Research as a Service (MRAS) resources to conduct Requests for Information (RFIs).

**Comment by jfredrickson** [comment]
The update is mostly ready, just waiting for a link to "Develop and issue the solicitation" that will be created in https://github.com/GSA/oasis-plus/issues/137.

**Comment by PowderD** [comment]
@klohman @rosamundtgov @vickimcfadden @jfredrickson Should we move the ["Register in Symphony" to the Market Research area? I'm having a hard time putting my mind around where to place this.  If customers do RFI's(aka market research) they will have to be registered in Symphony? Would we want to move it anywhere else in the process?

**Comment by klohman** [comment]
> @klohman @rosamundtgov @vickimcfadden @jfredrickson Should we move the ["Register in Symphony" to the Market Research area? I'm having a hard time putting my mind around where to place this. If customers do RFI's(aka market research) they will have to be registered in Symphony? Would we want to move it anywhere else in the process?

@PowderD If they need access earlier in the process than we have it noted, we should move it, but you know the process best so I'll leave it to you.

**Comment by PowderD** [comment]
Let's discuss on today's call.  Maybe talking it through will help.

*Jon DeWolfe  |  *
Program Manager Buyer Engagement
Services Marketplace
Federal Acquisition Service, U.S. General Services Administration
|(: 208.830.8517 | *: ***@***.***

**Comment by weiwang-gsa** [comment]
@klohman @rosamundtgov I noticed "Explore the guide below or view the [OASIS+ Buyers’ Ordering Guide (printable)" is removed from the 1st paragraph, and in issue [#449](https://github.com/GSA/oasis-plus/issues/449), Kristen you left comment to adjust the link to the one page ordering guide(printable), just want to double check if this sentence still remain or it should be gone?

**Comment by klohman** [comment]
@weiwang-gsa That sentence should remain - I don't think it needs be included until we have the printable ordering guide page to link to (so I didn't want it to block this one)

**Comment by weiwang-gsa** [comment]
@klohman sure, then I will merge it in and take care of conflicts in the other ticket.

**Comment by klohman** [comment]
@jfredrickson This one is ready for PSHC review (I'm not sure if it is merged into staging yet)

**Comment by klohman** [comment]
@ekidenda @PowderD This is ready for your review [here](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/)

**Comment by ekidenda** [comment]
@klohman I see that "Explore the guide below or view the [OASIS+ Buyers’ Ordering Guide (printable)](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/oasis-plus/buyers-guide/printable/)." is maintained in the page and I assume that what we need to review on this page. If so, it looks good unless @PowderD has more to offer - please check it too.

**Comment by klohman** [comment]
@ekidenda Hold for a second. There are some other updates from the copy doc we also need you to review, but I have some questions.

@rosamundtgov  
Copy doc has Manage the task order as a separate link under "Develop the solicitation and award the task order" and there are two pages for award the task order and then manage the task order. It looks like we combined them on this landing page, and we only link to award the task order. Is this intentional? I can't recall the reasoning

**Comment by rosamundtgov** [comment]
I think some of this got lost in translation. @ekidenda @PowderD If you're good with these updates, we will make them.

The Market Research section should include: 
 
-Title: Register in Symphony [link: oasis-plus/buyers-guide/register-in-symphony]
-Description: Familiarize yourself with Symphony and learn how to use it to complete market research. 

The Prepare to develop your solicitation section should include: 

- Title: Learn how to use Symphony [link: /resources/#proposal-submission-task-management]
- Description: Check out our Symphony resources; learn how to build submit proposals, manage task orders, and complete other tasks related to the buying process.

The Develop the solicitation and award the task order section should include: 

- Title: Award the task order [buyers-guide/award-task-order]
- Description: Learn about the factors and requirements involved in proposal evaluation and task order award consideration.

- Title: Manage the task order [buyers-guide/manage-task-order]
- Description: Understand the process of reassigning task orders for administration, quality assurance, UR subcontracting plans, performance evaluation, and task order closeout.

**Comment by PowderD** [comment]
@rosamundtgov @ekidenda This looks right to me, unless Esther has additional comments

**Comment by rosamundtgov** [comment]
@ekidenda @PowderD this is ready for your review: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/oasis-plus/buyers-guide//

**Comment by PowderD** [comment]
@rosamundtgov Ready to go!
